Added AdminAbsoluteUrlMixin

ar4s opened the pull request:
Added new mixin (`AdminAbsoluteUrlMixin`) for automatically generate URL
based on model.

xliiv commented on src/ralph/assets/models/mixins.py:
```diff
@@ -64,3 +60,11 @@ def save(self, update_last_seen=False, *args, **kwargs):
+    def get_absolute_url(self):
+        info = self._meta.app_label, self._meta.model_name
+        return reverse(
+            'admin:{}_{}_change'.format(*info), args=(self.pk,)
```
could it be simply:
`'admin:{}_{}_change'.format(self._meta.app_label, self._meta.model_name), args=(self.pk,)`
?

szok commented on src/ralph/assets/models/mixins.py:
```diff
@@ -1,9 +1,5 @@
+from django.core.urlresolvers import reverse
 from django.db import models
 from django.utils import timezone
 from django.utils.encoding import python_2_unicode_compatible
```
Delete this line

mkurek commented on src/ralph/assets/models/mixins.py:
```diff
@@ -64,3 +60,12 @@ def save(self, update_last_seen=False, *args, **kwargs):
+
+
+class AdminAbsoluteUrlMixin(object):
+    def get_absolute_url(self):
```
maybe some unit test? :smirk:

szok replied:
:+1:

Commit:
added AdminAbsoluteUrlMixin

    Added new mixin (``AdminAbsoluteUrlMixin``) for automatically generate URL
    based on model.

szok commented:
:ok:

mkurek commented:
:ok:
